package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "falconry.github.io",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "serve": "webpack serve --port 8000 --config webpack-dev.config.js",
+    "build": "webpack --config webpack-prod.config.js"
+  },
+  "dependencies": {
+    "bootstrap": "^4.5.3",
```

Comment:
I know bootstrap 5 is still in beta, but usually the bootstrap pre-release are still very stable.
Moving to v5 would also let us remove jquery

Reply:
Technically your right. This PR already includes a lot of changes and I don't have that many resources right now, so I would propsose merging it and upgrade Bootstrap later.